### PR TITLE
Add Clickhouse Insert Test

### PR DESCRIPTION
**Pull request by fpacifici**
Adds two tests for the load testers:
- a basic clickhosue query (that only runs SELECT 1)
- an insertion test that generates syntetic load according to a config file.

The insert test produces batches (one abtch per request) and sends
them to the Clickhouse HTTP interface.

Syntetic traffic is generated by a set of `dataproviders`. They include
providers like:
- random int
- random string 
- UUID
- sequence
- etc.

A configuration file in this form is provided, that defines the table schema
and assigns a dataprovider to each column:

```
{
    "name": "my test",
    "description": "my description",
    "testType": "clickhouseInsert",
    "attackDuration": "5m",
    "numMessages": 1,
    "per": "1s",
    "params": { 
        "batchSize": 100000,
        "tableName": "test_table2_dist",
        "config": {
            "seq": {
                "valueType": "partitionedSequence",
                "config": {}
            },
            "constStr": {
                "valueType": "const",
                "config": {
                    "value": "my_val"
                }
            },
            "id": {
                "valueType": "uuid",
                "config": {}
            },
...
```

### CODE REVIEW

File: web_server/worker.go [hunk 1]
```diff
@@ -281,7 +281,7 @@ func worker(targetUrl string, statsdAddr string, configParams *configParams, par
 		default:
 			if loadTester != nil {
 				rate := vegeta.Rate{Freq: params.NumMessages, Per: params.Per}
-				attacker := vegeta.NewAttacker(vegeta.Timeout(time.Millisecond*500), vegeta.Redirects(0), vegeta.MaxWorkers(1000))
+				attacker := vegeta.NewAttacker(vegeta.Timeout(time.Millisecond*500), vegeta.Redirects(0), vegeta.MaxWorkers(1))
```
[the remark this answers is not in the document]
Comment (fpacifici) [reply web_server/worker.go:284]: That was one experiment I did originally. 
I did not revert it in this PR as this one covers it https://github.com/getsentry/go-load-tester/pull/31

File: tests/clickhouseQuery_test.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package tests
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestUnmarshalJson(t *testing.T) {
+	jsonData := json.RawMessage(`{"multiplsier": 100}`)
```
Comment (nachivrn) [inline tests/clickhouseQuery_test.go:9]: there is a typo. mulitpier.

trying to understand how the test was able to unmarshall to type ClickhouseQueryJobRaw

File: tests/dataproviders/config.go [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+type ClickhouseInsertJobRaw struct {
+	BatchSize   int64                         `json:"batchSize"`
+	TableName   string                        `json:"tableName"`
+	Partitions  int                           `json:"partitions"`
+	PartitionId int                           `json:"partitionId"`
+	Config      map[string]ClickhouseFieldRaw `json:"config"`
+}
+
+func (raw ClickhouseInsertJobRaw) Into(result *ClickhouseInsertJob) error {
```
Comment (nachivrn) [inline tests/dataproviders/config.go:18]: Question: Why was receiver ClickhouseInsertJobRaw not passed by reference

**Review by nachivrn — APPROVED** [review]
Thank you @fpacifici for creating the load tester. It was a great help in testing within the sandbox.

File: tests/clickhouseQuery_test.go (hunk 2 above)
Comment (fpacifici) [reply tests/clickhouseQuery_test.go:9]: it was not. Thanks for pointing out.

File: tests/dataproviders/config.go (hunk 3 above)
Comment (fpacifici) [reply tests/dataproviders/config.go:18]: It did not. Changed.

**Pull request merged**